pytest_fixtures/component/leapp_client.py

```diff
@@ -0,0 +1,204 @@
+from broker import Broker
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import PRDS
+from robottelo.hosts import ContentHost
+from robottelo.logging import logger
+
+synced_repos = pytest.StashKey[dict]
+
+RHEL7_VER = '7.9'
+RHEL8_VER = '8.10'
+RHEL9_VER = '9.4'
+
+RHEL_REPOS = {
```

Comment:
I think these all should be available in constants or near to repos helper classes, please use from their.

Reply:
We don't have any use anywhere other than leapp so I kept in this module only. I feel that the constants common across all the module should be kept in constants file. Wdyt?